qt/scientific_interfaces/Indirect/IndirectFunctionBrowser/IqtFunctionModel.cpp

```diff
@@ -625,6 +628,34 @@ void IqtFunctionModel::estimateStretchExpParameters() {
   }
 }
 
+void IqtFunctionModel::estimateExpParameters() {
+  auto const heightName1 = getParameterName(ParamID::EXP1_HEIGHT);
+  auto const lifeTimeName1 = getParameterName(ParamID::EXP1_LIFETIME);
+  if (!heightName1 || !lifeTimeName1)
+    return;
+  auto const heightName2 = getParameterName(ParamID::EXP2_HEIGHT);
+  auto const lifeTimeName2 = getParameterName(ParamID::EXP2_LIFETIME);
```

Comment:
The check on L634 to see if heightName1 or lifeTimeName1 are empty needs extending to include the xxx2 variables as well. So you just need to move the if statement down a couple of lines and extend it